docs/topics/con_shared-messaging-work-queue-design-tradeoffs.adoc

```diff
@@ -0,0 +1,17 @@
+[id='con_shared-messaging-work-queue-design-tradeoffs_{context}']
+
+= {name-mission-shared-messaging-work-queue} design tradeoffs
+
+.Design Tradeoffs
+[width="100%",options="header"]
+|====================================================================
+|Pros           |Cons
+a|
+* Modularity: due to the decoupling between the workers and the central component of your messaging infrastructure, you can swap out the central component depending on which functionalities you need it to provide.
+For example, using an interconnect router instead of the broker sacrifices message persistence while allowing faster processing time per message.
+* Reliability: Using a broker with persistent storage prevents loss of messages in case a target worker module becomes unavailable. The router persists the message and forwards it to a different available worker.
```

Comment:
"worker module" -> "worker process"?

Reply:
Also probably shouldn't use the word "router" here, because it's mentioned above in conjunction with "sacrifices persistence". I believe you wanted to use "broker" here.

Reply:
True. Thank you for catching this. I'll correct the term:)

Reply:
fixed:)